symbology/render-multilayer-symbols/src/main/java/com/esri/samples/render_multilayer_symbols/RenderMultilayerSymbolsSample.java

```diff
@@ -0,0 +1,459 @@
+      double offset = 20;
+
+      // create labels to go above each category of graphic
+      addTextGraphics();
+      
+      // create picture marker symbols, from URI or embedded resources
+      PictureMarkerSymbolLayer pictureMarkerFromUri = new PictureMarkerSymbolLayer((
+        "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Recreation/FeatureServer/0/images" +
+          "/e82f744ebb069bb35b234b3fea46deae"));
+      addGraphicFromPictureMarkerSymbolLayer(pictureMarkerFromUri, 0);
+      
+      PictureMarkerSymbolLayer pictureMarkerFromImage = new PictureMarkerSymbolLayer(
+        new Image(Objects.requireNonNull(getClass().getResourceAsStream("/blue_pin.png")), 
+          0, 50, true, true));
+
+      addGraphicFromPictureMarkerSymbolLayer(pictureMarkerFromImage, offset);
+
+      // add graphics with simple vector marker symbol elements (MultilayerPoint Simple Markers on app UI)
+      var solidFillSymbolLayer = new SolidFillSymbolLayer(ColorUtil.colorToArgb(Color.RED));
+      var multilayerPolygonSymbol = new MultilayerPolygonSymbol(List.of(solidFillSymbolLayer));
+      var solidStrokeSymbolLayer = new SolidStrokeSymbolLayer(1, ColorUtil.colorToArgb(Color.RED),
+        List.of(new DashGeometricEffect()));
+      var multilayerPolylineSymbol = new MultilayerPolylineSymbol(List.of(solidStrokeSymbolLayer));
+      
+      // define vector element for a diamond, triangle and cross
+      var diamondGeometry = Geometry.fromJson("{\"rings\":[[[0.0,2.5],[2.5,0.0],[0.0,-2.5],[-2.5,0.0],[0.0,2.5]]]}");
+      var triangleGeometry = Geometry.fromJson("{\"rings\":[[[0.0,5.0],[5,-5.0],[-5,-5.0],[0.0,5.0]]]}");
+      var crossGeometry = Geometry.fromJson("{\"paths\":[[[-1,1],[0,0],[1,-1]],[[1,1],[0,0],[-1,-1]]]}");
+      
+      addGraphicsWithVectorMarkerSymbolElements(multilayerPolygonSymbol, diamondGeometry, 0);
+      addGraphicsWithVectorMarkerSymbolElements(multilayerPolygonSymbol, triangleGeometry, offset);
+      addGraphicsWithVectorMarkerSymbolElements(multilayerPolylineSymbol, crossGeometry, 2 * offset);
+      
+      // TODO: see if it's possible to cut down the lines of code as I've demonstrated above for the remaining methods!
+
+      // create line marker symbols
+      addLineGraphicsWithMarkerSymbols(List.of(4.0, 6.0, 0.5, 6.0, 0.5, 6.0), 0);
+      addLineGraphicsWithMarkerSymbols(List.of(4.0, 6.0), offset);
+      addLineGraphicsWithMarkerSymbols(List.of(7.0, 9.0, 0.5, 9.0), 2 * offset);
+
+      // create polygon marker symbols
+      addPolygonGraphicsWithMarkerSymbols(List.of(-45.0,45.0),0);
+      addPolygonGraphicsWithMarkerSymbols(List.of(-45.0), offset);
+      addPolygonGraphicsWithMarkerSymbols(List.of(90.0), 2 * offset);
+
+      // define vector element for a hexagon which will be used as the basis of a complex point
+      var complexPointGeometry = Geometry.fromJson("{\"rings\":[[[-2.89,5.0],[2.89,5.0],[5.77,0.0],[2.89,-5.0],[-2.89,-5.0],[-5.77,0.0],[-2.89,5.0]]]}");
```

Comment:
let's splite this one over two lines

```suggestion
      // define vector element for a hexagon which will be used as the basis of a complex point
      // define vector element for a hexagon which will be used as the basis of a complex point
      var complexPointGeometry = Geometry.fromJson("{\"rings\":" +
        "[[[-2.89,5.0],[2.89,5.0],[5.77,0.0],[2.89,-5.0],[-2.89,-5.0],[-5.77,0.0],[-2.89,5.0]]]}");
```